### PR TITLE
BAH-436 Add link to the admin page from the legacy admin UI

### PR DESCRIPTION
**Pull request by sanishmaharjan**
[BAH-436](https://bahmni.atlassian.net/browse/BAH-436): Link to the new admin page from the legacy admin UI
Please code review.
@mks-d

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -101,6 +101,15 @@
         <scope>provided</scope>
       </dependency>
       <!-- End OpenMRS core -->
+
+      <dependency>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>legacyui-omod</artifactId>
+        <version>1.0</version>
```
Comment (mks-d) [inline pom.xml]: Create a property `<legacyuiVersion>1.0</legacyuiVersion>` to maintain the version with the others, see [here](https://github.com/openmrs/openmrs-module-bedmanagement/blob/28fb48c8a019e9257838f75e9aa61c3cb2d5cfc9/pom.xml#L38-L46).

File: pom.xml [hunk 2]
```diff
@@ -101,6 +101,15 @@
...
+        <type>jar</type>
+        <scope>provided</scope>
+      </dependency>
```
Comment (mks-d) [inline pom.xml]: Neither `<type>jar</type>` nor `<scope>provided</scope>` are really needed here. Can you check at runtime?

File: omod/pom.xml [hunk 3]
```diff
@@ -89,6 +89,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>legacyui-omod</artifactId>
+			<type>jar</type>
```
Comment (mks-d) [inline omod/pom.xml]: The `<type/>` tag is not necessary, however you should add `<scope>provided</scope>` here. Can you check at runtime?

File: pom.xml (hunk 2 above)
Comment (sanishmaharjan) [reply pom.xml]: @mks-d done.

**Pull request merged**